src/Microsoft.DotNet.XHarness.iOS.Shared/Listeners/SimpleListenerFactory.cs

```diff
@@ -61,6 +64,9 @@ public class SimpleListenerFactory : ISimpleListenerFactory
 
         switch (transport)
         {
+            case ListenerTransport.Relay:
```

Comment:
This seems like the `case ListenerTransport.Tcp:`. What's the actual functional effect of passing `relay = true` in here? Shouldn't we do something special with the listener then?